docker-images/jmxtrans/docker-entrypoint.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -e
+set +x
```

Comment:
Do you think there's some debug value in being able to conditionally echo the commands executed in the preparation, e.g. if some env var is set?

Reply:
I don't think I see the need. I guess it could be aded later if we find it missing In many containers it was anyway disabled already before this PR, so it is more cleanup then radical change.

Reply:
@tombentley So, is it ok with you like this? Or do you want some changes?